static/src/js/utils/checkForCMRFetchDraftLag.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Compares the revision Ids to ensure the draft retched is the most current
+ * @param {String} fetchedRevisionId The revision Id that comes back from CMR
+ * @param {String} expectedRevisionId The revision Id that was set on ingest (expected revision)
+ */
+
+const checkForCMRFetchDraftLag = (fetchedRevisionId, expectedRevisionId) => {
+  if ((fetchedRevisionId && expectedRevisionId) && (fetchedRevisionId !== expectedRevisionId)) {
+    throw new Error('Delay in CMR has detected. Refresh the page in order to see latest revision')
```

Comment:
I almost feel like you ought to put the revision id on the page somewhere.   How will the user know when they refresh if they got the latest revision id?   Refreshing won't really fix it if CMR is still lagging.   Might be worth talking to John about how we should let the user know there is a problem and wording to go along with that.